applications/DAIN/networks/dain_slowmotion.py

```diff
@@ -0,0 +1,148 @@
+import paddle.fluid as fluid
+import resblock
+import time
+import pwcnet
+
+
+class DAIN_slowmotion(fluid.dygraph.Layer):

```

Comment:
I noticed that "class DAIN_slowmotion(fluid.dygraph.Layer):" is 1.x dygraph mode，so I'm not sure whether it should change to paddle.nn.Layer or not, and other fluid.xx APIs as well.

Reply:
@Joejiong Thanks!  We can upgrade all code when 2.0 API is ready,